src/utils/i18n/enzymeHelper.jsx

```diff
@@ -32,10 +32,11 @@ export function shallowWithIntl(node, { context, disableLifecycleMethods } = {})
   );
 }
 
-export function mountWithIntl(node, { context, childContextTypes } = {}) {
+export function mountWithIntl(node, { attachTo, context, childContextTypes } = {}) {
```

Comment:
What does attachTo do? This is the only place I see it referenced, and I don't see any additional parameters to the use of mountWithIntl in the codebase. Is it still needed?

Reply:
I briefly tried using the [attachTo](http://airbnb.io/enzyme/docs/api/ReactWrapper/detach.html) option in Enzyme, but this function wasn't passing that option through to Enzyme's `mount` function without this change. I decided to keep this here even though I stopped using `attachTo` in case we ever needed to use it in the future (otherwise we might have to go through the same confusion about why the option isn't going through).